pydoctor/templatewriter/util.py

```diff
@@ -17,3 +18,22 @@ def templatefile(filename):
 
 def templatefilepath(filename):
     return FilePath(templatefile(filename))
+
+
+class Page(Element):
+
+    def __init__(self, system: model.System):
+        super().__init__()
+        self.system = system
+
+    @property
+    def project_tag(self) -> Tag:
+        system = self.system
```

Comment:
What is the project_tag ?

And then we also have project_link somewhere in the code.

I think that at the view-model level we should only present raw data and then it's up to the template to decide how to present it... in an A tag or a button with JS actions :)

Reply:
The old situation was a mess and we can't really clean it up without breaking compatibility. Since #299 has to break compatibility anyway, I think it's best to do cleanups there and after that has been merged.

The mess was as follows:
- a slot named `project` that was used by one template
- a renderer named `project` that was used by three templates
- a renderer named `project_link` that is not used by pydoctor, but is used by Twisted

The `project_tag` property is an attempt to make a single implementation of rendering a project name that links to the project URL that can be used by all of the three listed above.